src/session/sessionActions.js

```diff
@@ -37,11 +30,7 @@ export const appOnline = (isOnline: boolean) => (
   }
 };
 
-export const appState = (isActive: boolean) => (
-  dispatch: Dispatch,
-  getState: GetState,
-  // $FlowFixMe
-): ?AppStateAction => {
+export const appState = (isActive: boolean) => (dispatch: Dispatch, getState: GetState): void => {
```

Comment:
The type of the first function will be something like `Thunk<AppStateAction>` (if we imagine we have declared `Thunk` already)

Reply:
Do you mean we should annotate this in a new commit?